packages/rpc-core/src/rpc-subscriptions/__typetests__/slots-updates-notifications-type-test.ts

```diff
@@ -0,0 +1,33 @@
+/* eslint-disable @typescript-eslint/ban-ts-comment */
+
+import { RpcSubscriptions } from '@solana/rpc-transport/dist/types/json-rpc-types';
+
+import { Slot } from '../../rpc-methods/common';
+import { UnixTimestamp } from '../../unix-timestamp';
+import { SolanaRpcSubscriptions, SolanaRpcSubscriptionsUnstable } from '../index';
+
+async () => {
+    const rpcSubscriptions = null as unknown as RpcSubscriptions<
+        SolanaRpcSubscriptions & SolanaRpcSubscriptionsUnstable
```

Comment:
You should _only_ include the relevant one.
```suggestion
        SlotsUpdatesNotificationsApi
```